src/Groups/gsets.jl

```diff
@@ -931,13 +931,79 @@ maximal_blocks(G::GSet) = error("not implemented")
 minimal_block_reps(G::GSet) = error("not implemented")
 all_blocks(G::GSet) = error("not implemented")
 
+
+"""
+    is_transitive(Omega::GSet)
+
+Determine whether the group associated with `Omega` acts transitively.
+
+# Examples
+```jldoctest
+julia> G = sylow_subgroup(symmetric_group(6), 2)[1]
+Permutation group of degree 6 and order 16
+
+julia> Omega = gset(G);
+
+julia> is_transitive(Omega)
+false
+```
+"""
+function is_transitive(Omega::GSet)
+    return length(orbits(Omega)) == 1
+end
 function is_transitive(Omega::GSetByElements)
     length(Omega.seeds) == 1 && return true
     return length(orbits(Omega)) == 1
 end
 
+"""
+    is_regular(Omega::GSet)
+
+Return whether the group action associated with `Omega`
+is regular (i.e., transitive and semiregular).
+
+# Examples
+```jldoctest
+julia> G = symmetric_group(6);
+
+julia> H = sub(G, [G([2, 3, 4, 5, 6, 1])])[1]
+Permutation group of degree 6
+
+julia> OmegaG = gset(G);
+
+julia> OmegaH = gset(H);
+
+julia> is_regular(OmegaH)
+true
+
+julia> is_regular(OmegaG)
+false
+```
+"""
 is_regular(Omega::GSet) = is_transitive(Omega) && length(Omega) == order(acting_group(Omega))
 
+"""
+    is_semiregular(Omega::GSet)
+
+Return whether the action of `G` on `L` is semiregular
+(i.e., the stabilizer of each point is the identity).
```

Comment:
The three docstrings use three different ways to express something similar :-). I think this one is worse than the other two because it references `G` and `L` without stating what they are.

I suggest we come up with a common phrase and use that for all of these docstrings. 

Minimalistic:
```suggestion
Return whether `Omega` is semiregular, i.e.,
the stabilizer of each point is the identity.
```
A bit more:
```suggestion
Return whether the group `G` acting on `Omega` in fact acts
semiregular, i.e., the stabilizer of each point is the identity.
```
Or something else...

Reply:
Okay, I made some minor tweaks and think it is much more consistent now